webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -2961,7 +2986,15 @@ var userOverride = {
                     'aria-label',
                     LEAD_COLLECTION_LABEL.submit
                 );
-                leadCollectionHeading.innerHTML = LEAD_COLLECTION_LABEL.heading;
+                if (appOptions.preLeadCollection) {
+                    leadCollectionHeading.innerHTML =
+                        LEAD_COLLECTION_LABEL.heading;
+                } else {
+                    leadCollectionHeading.innerHTML = appOptions.appSettings
+                        .chatWidget.preChatGreetingMsg
+                        ? appOptions.appSettings.chatWidget.preChatGreetingMsg
+                        : '';
```

Comment:
@devashishmamgain will it be better if instead of showing empty, we show our default msg?

Reply:
it is a mandatory thing in dashboard so its fine. check is just for the purpose of avoiding any null or error